priv/app/call_tracer.jsx

```diff
@@ -22,47 +123,34 @@ export default class CallsTracer extends React.Component {
     clearTimeout(this.timeoutRef);
   }
 
-  handleClick(e) {
-    e.preventDefault();
+  handleCaptureStart() {
+    let fun = this.props.fun;
+    let threshold = this.state.threshold_value || this.props.defaultThreshold;
+    let limit = this.state.limit_value || this.props.defaultLimit;
 
-    var fun = this.props.fun;
-    var threshold = $(React.findDOMNode(this.refs.thresholdInput)).val();
-    var limit = $(React.findDOMNode(this.refs.limitInput)).val();
+    this.setState({ status: this.Status.RUNNING });
 
     $.ajax({
       url: "/api/capture",
       data: {
         mod: fun[0], fun: fun[1], arity: fun[2],
         threshold: threshold,
-        limit: limit
-      }
-    }).success(function(e) {
-      this.state.capture_id = e.capture_id;
+        limit: limit }
+    }).done((response) => {
+      this.state.capture_id = response.capture_id;
       this.state.offset = 0;
+      this.state.items = [];
       this.setState(this.state);
       this.getCaptureData();
-
-    }.bind(this));
+    });
   }
 
-  handleRowExpandClick(ref, e) {
-    e.preventDefault();
-
-    var target_row = React.findDOMNode(this.refs[ref]);
-
-    if ($(target_row).data("expanded")) {
-      $(target_row).data("expanded", false);
-      $(target_row).removeClass("row-expanded");
-      $(target_row).addClass("row-normal");
-      $(".expand-chevron", target_row).removeClass("glyphicon-chevron-down");
-      $(".expand-chevron", target_row).addClass("glyphicon-chevron-right");
-    } else {
-      $(target_row).data("expanded", true);
-      $(target_row).removeClass("row-normal");
-      $(target_row).addClass("row-expanded");
-      $(".expand-chevron", target_row).removeClass("glyphicon-chevron-right");
-      $(".expand-chevron", target_row).addClass("glyphicon-chevron-down");
-    }
+  handleCaptureStop() {
+    let fun = this.props.fun;
```

Comment:
I think that I introduced this to the code when xprof was meant to be a simple _function_ tracer.